api/utils/token_validation.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework.authentication import get_authorization_header
+from users.models import TokenBlacklist
+from rest_framework import permissions, status
+from rest_framework.response import Response
+
+
+class TokenAllowedPermission(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        auth = get_authorization_header(request).split()
+        token = auth[1]
+        try:
+            blacklisted = TokenBlacklist.objects.get(token=token)
+            if blacklisted != []:
```

Comment:
Hello @IssaIan ,
Instead of using checking if the blacklisted list is empty this way, just use:
```
if blacklisted:
   do-the-operations
```
An empty list, dictionary and tuple are false.
You can read more on this from [Andela Python Best Practices](https://github.com/andela/bestpractices/wiki/Python-Best-Practices)